.github/workflows/changesets-status.yml

```diff
@@ -13,19 +13,6 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version-file: ".nvmrc"
-      - name: Cache node modules
-        uses: actions/cache@v3
-        env:
-          cache-name: cache-node-modules
-        with:
-          path: ~/.npm
-          key: ${{ runner.os }}-qa-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
-          restore-keys: |
-            ${{ runner.os }}-qa-${{ env.cache-name }}-
-            ${{ runner.os }}-qa-
-            ${{ runner.os }}-
-      - name: Install deps
-        run: npm ci
       - name: Check changesets status
-        run:  npm run change:status -- --since origin/main
+        run: npx @changesets/cli status --since origin/main
```

Comment:
Are we sure we want to version to be mutable? It may be better to lock the version and hashes using `packages.json`